handler/__tests__/action_current_news.js

```diff
@@ -8,7 +8,7 @@ describe("action_current_news", () => {
     // e4d4c2941dd54f549393e9c3384e2d10900d36c7
     const chat = new facebook.Chat();
     return current_news(chat).then(() => {
-      new Expect(chat).buttons(
+      new Expect(chat).request().buttons(
```

Comment:
Are you required to always use the expect library with a valid chat object? Even if you just want to check requests?
Why is the request expect bare?!
Is it even necessary for this test to require that a request was made? I though as long as the chat output was correct, it doesn't matter how the function got to that. Consider e.g.: Somebody implements a cache for the requests. This is perfectly valid, but the test would fail.

Reply:
You only need to pass the `chat` object if you want to expect Facebook API calls, which will fail if you don't pass it.
We will remove the `.request()` expectation, it was just a test if the test works. It can be used without a parameter to accept any request.